build.cake

```diff
@@ -130,6 +132,68 @@ void DoArchive(string runtime, DirectoryPath inputFolder, FilePath outputFile)
 }
 
 /// <summary>
+///  Generate the scripts which target the OmniSharp binaries.
+/// </summary>
+/// <param name="outputRoot">The root folder where the publised (or installed) binaries are located</param>
+void CreateRunScript(string outputRoot, string scriptFolder)
+{
+    if (IsRunningOnWindows())
+    {
+        var desktopScript = $"{scriptFolder}/OmniSharp.cmd";
+        var coreScript = $"{scriptFolder}/OmniSharp.Core.cmd";
```

Comment:
I just played around with System.IO.Path.Combine and unfortunately Cake uses `/` on Windows.
This results in weird file paths when combining the two, like `C:/Work/coreclr/omnisharp-roslyn\artifacts\publish\OmniSharp\default\`. I will try to get rid of all the Cake paths used by the script, but this should be kept in mind for future work as-well.